exercises/resistor-colors/examples/success-standard/package.yaml

```diff
@@ -0,0 +1,16 @@
+name: exercism-resistor-colors
```

Comment:
Since it's not necessary to prepend `exercism-` to the name, I think we should prefer not to do so, and only do so when it is necessary (as it is in `clock`, the only exercise that does so)

Reply:
Changed.